core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/jdk/JdkAggregators.java

```diff
@@ -115,8 +115,7 @@ public final class JdkAggregators {
 
 	// VM Info
 	public static final IAggregator<String, ?> JVM_NAME = distinctAsString(VM_INFO, JdkAttributes.JVM_NAME);
-	public static final IAggregator<IQuantity, ?> JVM_PID = min(JdkAttributes.JVM_PID.getName(), null, VM_INFO,
```

Comment:
Part of the public API. Perhaps better to deprecate?

Reply:
Do you mean to use @deprecate tag here instead of deleting it completely?

Reply:
Yeh otherwise, this would be a breaking change for consumer of the flight recorder libraries.

Reply:
Precisely. We can remove it in JMC 10. :)